src/main/java/org/rus4j/numbify/lang/ru/Russian.java

```diff
@@ -30,6 +30,22 @@ public Russian(RuDeclension declension, Gender[] genders, Currency currency, Str
         this.decimalSeparator = decimalSeparator;
     }
 
+    /**
+     * 'Russian' constructor with default genders.
+     * By default, genders[0] = Gender.MALE, genders[1] = Gender.FEMALE.
+     */
```

Comment:
No need to document secondary constructors. JavaDoc for main (full) constructor is enough